documentation/quickstart/quickstart_gha.md

```diff
@@ -54,7 +56,11 @@
    
    ![image](./images/gh-generate.png)
 
-   2.3. Go to your Github organization and create an **empty** repo. This is going to be the repo, into which you'll push your sparse checkout local repo. (more to that later) 
+   2.3. Go to your Github organization and create an empty repo. This is going to be the repo into which you'll push your sparse checkout local repo. (more to that later).
```

Comment:
@sdonohoo in the new sparse checkout script I hope to use, I've removed the need to do this manually by using the github CLI. This however means that the user would need to install the github CLI though... The command is as follows:`gh repo create $project_name --private`

Reply:
Thanks, @cindyweng. So this section of doc will simply be deleted. The user will specify the github repo name in sparse_checkout.sh without creating it and it will be created for them during the script execution. Correct?